### ISSUE TITLE
Adding support for Odroid C2

### ISSUE THREAD

**Issue by Skinah** [openhab/openhabian#432]
Hi all,

Looking for some feedback and discussion before I make a pull request...

Recently I have been playing with getting an Odroid C2 running with openhab2 and found the zulu java will not install with the repo method and needs to use the "archive" method. The recommended repo simply reports that is does not support arm64 when used on Ubuntu minimal 18.04 LTS

Because of this Openhabian fails to install unless you select Oracle java. The output when it fails is..

```
$ apt -y install zulu-embedded-8
Reading package lists... Done
Building dependency tree
Reading state information... Done
E: Unable to locate package zulu-embedded-8
FAILED
```

I have discussed this on the openhab forum (link below) and Markus has kindly assisted with some info so I have created some minor changes which are found in the below fork...
https://github.com/openhab/openhabian/compare/master...Skinah:master

There are a number of changes I would like to discuss on top of what is in the fork and they are:

1. Raw image for the odroid only has a single user "root". Would be good to auto create a new 'everyday user' which does not appear to happen with my testing as the openhabian-setup script simply asks you on first run what user you already have.

2. A check for warning the user that their root account still needs the password changed from the default 'odroid' value.

3. Manual/fresh setup (option 60) does not offer the manual 'zulu archive' method to install java. Could this be added. There will be users wanting this with Odoroid XU4, and many other arm64 devices not just C2 owners. 

Discussion on forum is here.
https://community.openhab.org/t/how-to-easily-install-openhab2-on-odroid-c2-arm64/49194

**Comment by EliasGabrielsson** [comment]
Hi again! Have read up some more and have some input.

First of all if we have the intention of supporting a new board we shall do it completely and firmly for all feature set. The end user shall expect the same experience for openhabian regardless of hardware if it's marked as `supported`. Remember the mantra of openhabian: **A home automation enthusiast doesn't have to be a Linux enthusiast!**

That means in practices that we shall create a build-script (`openhabian/build.sh`) producing Odroid C2 image. This not just "extra work" it will solve issues like question number 1 and 2 above as this the environment will be close to other openhabian installation.  Documentation will also be correct.

* Base image of Ubuntu 18.04 minimal image: https://wiki.odroid.com/odroid-c2/os_images/ubuntu/minimal_image
* Create a new section "Odroid C2" in https://github.com/openhab/openhabian/blob/master/build.sh applying hostname changes, injecting "boot-script" and so on..

This maybe feels cumbersome but this will make code easy to maintain and support in the long run.
Do you feel that you have time for this challenge?

**Comment by EliasGabrielsson** [comment]
I get your point. It's all boils down to what is expected by `supported`. So far a supported board means 100% compatibility with all scripts which take some effort to reach. BUT that doesn't stop effort of doing openhabian better. Broader plattform detection will be included in v1.5 which solves some of the issues you are experience. 

I also stumbled upon the project [Armbian](https://www.armbian.com/) which maybe could solve many issues with board support. Will look into it when times allow.

**Comment by EliasGabrielsson** [comment]
Superseded by this idea: https://github.com/openhab/openhabian/issues/622
